### PR TITLE
add case for unballanced src and dst and fix code

### PR DESCRIPTION
**Pull request by a-bougouin**
Fix an unexpected case for unballanced src and dst for unified values (see tests).

### CODE REVIEW

**Comment by a-bougouin** [comment]
I see, I'll try

**Comment by a-bougouin** [comment]
@mrasu done, thanks!

File: lib/wovnrb/html_replacers/html_text_replacer.rb [hunk 1]
```diff
@@ -107,17 +107,21 @@ def data_to_node(str)
     # @param [Nokogiri::XML::Node] node_1 The first node to align.
     # @param [Nokogiri::XML::Node] node_2 The second node to align.
     def align_nodes(node_1, node_2)
-      base_node = (node_1.children.count > node_2.children.count) ? node_1 : node_2
-      node_to_adjust = (base_node == node_1) ? node_2 : node_1
+      ajust_node_alignment(node_1)
```
Comment (mrasu) [inline lib/wovnrb/html_replacers/html_text_replacer.rb]: Do we need to add empty TextNode when both nodes don't have TextNode at first?

Comment (a-bougouin) [inline lib/wovnrb/html_replacers/html_text_replacer.rb]: ^^, good point, I'fix that

Comment (a-bougouin) [inline lib/wovnrb/html_replacers/html_text_replacer.rb]: done, thanks!

**Pull request merged**